src/Tasks/LegalProposition/PrintIllegal.hs

```diff
@@ -37,7 +37,7 @@ subTreeIllegal ::Bool -> SynTree BinOp Char -> String -> Gen String
 subTreeIllegal notFirstLayer (Binary operator a b) usedLiterals =
     allocateBugToSubtree notFirstLayer a b usedLiterals operator
 subTreeIllegal _ (Not a) usedLiterals = do
-    left <- ifUseIllegal True True a usedLiterals
+    left <- ifUseIllegal True True a usedLiterals -- no-spell-check
```

Comment:
> Is that because of the `True True`?

Yes. 

> 
> Shouldn't that be handled by
> 
> https://github.com/fmidue/logic-tasks/blob/5cdede4321e06a0a589ca913f5629a16e4af4e6a/.github/actions/spelling/patterns.txt#L3-L8
> 
> ?

It apparently did not work [here](https://github.com/fmidue/logic-tasks/actions/runs/8325078574). 
[It works](https://github.com/fmidue/logic-tasks/actions/runs/8325216979) when ignoring the line.

Besides that: There seems to be a problem with the workflow as you can see [here](https://github.com/fmidue/logic-tasks/actions/runs/8325078574/job/22777968704#step:2:231).

Reply:
About the problem with the workflow itself, can you open an issue for it? Not sure whether it is a permanent failure or something flukey.